src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+
+        int index = new Random().nextInt(Color.values().length);
+        return Color.values()[index];
     }
```

Comment:
```suggestion
    private Random random = new Random();
    public Color getRandomColor() {
        int randomIndex = random.nextInt(Color.values().length);
        return Color.values()[index];
    }
```

lets make Random as class level variable, not to create new instance of Random class every time we call getRandomColor() method